### ISSUE TITLE
Argument parsing is still broken in 6.7.0 (-r works but --require doesn't)

### ISSUE THREAD

**Issue by SystemParadox** [fgnass/node-dev#258]
Probably a continuation of #236.

In 5.2.0:

```
node-dev --no-notify --require source-map-support/register lib/server/index.js
// Correct source-mapped line numbers
```

In 6.7.0:

```
node-dev --no-notify --require source-map-support/register lib/server/index.js
// Wrong line numbers
node-dev --no-notify -r source-map-support/register lib/server/index.js
// Correct source-mapped line numbers
```

**Comment by bjornstar** [comment]
Your title says `--require` is broken and `-r` works.

However, your text here reads like `--require` works but `-r` is broken.

I'll take a look at it, but I'd like to understand which one is actually broken.

**Comment by SystemParadox** [comment]
Sorry I mixed up the comments. The title is correct: `-r` works but `--require` is broken. It looks like `-r` was fixed in #236, but not `--require` for some reason.

Thanks!
